### ISSUE TITLE
help

### ISSUE THREAD

**Issue by adriyunn** [hellowworld1337/ApexLegends-cheat#1]
can i have binary compiled pls

**Comment by Inverse25k** [comment]
it crashes lol

**Comment by Inverse25k** [comment]
https://github.com/pa1nz0r1337/ApexLegends-cheat/releases/download/apex/ApexInternal.dll

**Comment by joselitobkn9** [comment]
Guys, what injector are you using?

**Comment by Inverse25k** [comment]
Recoded face injector

**Comment by joselitobkn9** [comment]
Yes, I do not think it is a matter of the injector, if it crashes it is because it is outdated, you will have to update it, I read that :(

**Comment by Inverse25k** [comment]
sorry for late response but the reason why it crashes is 
![image](https://user-images.githubusercontent.com/79144625/139710371-71ab34ee-5294-41d3-a8e0-6869324a049c.png)

**Comment by LimLLL** [comment]
> sorry for late response but the reason why it crashes is ![image](https://user-images.githubusercontent.com/79144625/139710371-71ab34ee-5294-41d3-a8e0-6869324a049c.png)

Does this cheat still work?

**Comment by Inverse25k** [comment]
believe these offs are outdated
